packages/grid/src/vaadin-grid.d.ts

```diff
@@ -299,17 +299,31 @@ export interface GridEventMap<TItem> extends HTMLElementEventMap, GridCustomEven
  *
  * The following shadow DOM parts are available for styling:
  *
- * Part name | Description
- * ----------------|----------------
- * `row` | Row in the internal table
- * `cell` | Cell in the internal table
- * `header-cell` | Header cell in the internal table
- * `body-cell` | Body cell in the internal table
- * `footer-cell` | Footer cell in the internal table
- * `details-cell` | Row details cell in the internal table
- * `focused-cell` | Focused cell in the internal table
- * `resize-handle` | Handle for resizing the columns
- * `reorder-ghost` | Ghost element of the header cell being dragged
+ * Part name          | Description
+ * -------------------|----------------
+ * `row`              | Row in the internal table
+ * `expanded`         | Expanded row
+ * `selected`         | Selected row
+ * `details-opened`   | Row with details open
+ * `header`           | Header row
+ * `footer`           | Footer row
+ * `odd`              | Odd row
+ * `first`            | The first body row
+ * `last`             | The last body row
```

Comment:
Should this also have `-row` suffix (like the cells below)?

Reply:
Good point, will add it.

Reply:
Should it be `dragover-above-row` / `dragover-on-top-row`?

Reply:
Done.